examples/tts/conf/univnet/univnet.yaml

```diff
@@ -0,0 +1,79 @@
+name: "UnivNet"
+train_dataset: ???
+validation_datasets: ???
+
+defaults:
+  - model/generator: c16
+  - model/train_ds: train_ds
+  - model/validation_ds: val_ds
+
+model:
+  discriminator:
+    mpd:
+      periods: [2,3,5,7,11]
+      kernel_size: 5
+      stride: 3
+      use_spectral_norm: false
+      lrelu_slope: 0.2
+    mrd:
+      resolutions: [[1024, 120, 600], [2048, 240, 1200], [512, 50, 240]] # (filter_length, hop_length, win_length)
+      use_spectral_norm: false
+      lrelu_slope: 0.2
+  preprocessor:
+    _target_: nemo.collections.asr.parts.preprocessing.features.FilterbankFeatures
+    dither: 0.0
+    frame_splicing: 1
+    nfilt: 100
+    highfreq: 11025
```

Comment:
For sample rate=22050, you need to use highfreq=8000. @L0SG

Reply:
OK, and should we opt to use `nfilt=80` instead of `nfilt=100` for the default 22050Hz config? Considering the default one should work out of the box for the existing acoustic models @Oktai15 @blisc

Reply:
updated the default config with `nfilt=80` and `highfreq=8000`.